### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_New Project Finish_Keyboard: "More Settings" button is not accessible through keyboard

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3330]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.1.1

**NOTE: Same issue is observed in all the pop-ups containing similar control**

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Enter the required details, tab till "Next" button and press enter
13) Check if "More settings" button is accessible through keyboard

**Actual Result:**
"More Settings" button is not accessible through keyboard

**Expected Result:**
"More Settings" button should be accessible through keyboard

**User Impact:**
Keyboard only users and screen reader users will not be able to access the controls present under "More settings"

**MAS Reference:**
[MAS 2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
)

[Bug_6_IntelliJ_Customize_Keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451225/Bug_6_IntelliJ_Customize_Keyboard.pptx)
![Bug_6_IntelliJ_Deploy to Azure_Keyboard](https://user-images.githubusercontent.com/41108985/62204859-b7721f80-b3ab-11e9-9897-b2787c1cff6f.png)
![Bug_6_IntelliJ_Edit Configuration_Keyboard](https://user-images.githubusercontent.com/41108985/62204860-b7721f80-b3ab-11e9-9a6a-3a51c9780c35.png)

**Comment by Flanker32** [comment]
ActionButton & HideableDecorator cannot get focus using tab, both of them are components belong to IntelliJ which is out of the scope of out toolkits, correspond issues are tracking here https://youtrack.jetbrains.com/issue/IDEA-191322.

**Comment by Flanker32** [comment]
- Importance of experience: Low, it's not a common case to edit slot or before launch goal setting.
- Quality of experience: not impacted. Users could expand the slot panel by workaround mentioned above, and they could execute before launch goal in terminal manually if necessary, so this will not block keyboard only users
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@soumyatippapuram Please help us review this issue.

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us verify it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749759
